File: module1/.bài_tập_về_nhà/Customer.js
let Customer= function () {
    this.setName = function (name) {
        this.name = name;
    };
    this.getName = function () {
        return this.name
    };
    this.setCmnd = function (cmnd) {
        this.cmnd = cmnd;
    };
    this.getCmnd = function () {
        return this.cmnd;
    };
    this.setBirthday = function (birthday) {
        this.birthday = birthday;
    };
    this.getBirthday = function () {
        return this.birthday;
    };
    this.setEmail = function (email) {
        this.email = email;
    };
    this.getEmail = function () {
        return this.email;
    };
    this.setAddress = function (address) {
        this.address = address;
    };
    this.getAddress = function () {
        return this.address;
    };
    this.setTypecustomer = function (typecustomer) {
        this.typecustomer = typecustomer;
    };
    this.getTypecustomer = function () {
        return this.typecustomer;
    };
    this.setTypeservice = function (typeservice) {
        this.setTypeservice = typeservice;
    };
    this.getTypeService = function () {
        return this.typeservice;
    };
    this.setDiscount = function (discount) {
        this.discount = discount;
    };
    this.getDiscount = function () {
        return this.discount;
    };
    this.setQuantytiIncuded = function (quantytiIncuded) {
        this.quantytiIncuded = quantytiIncuded;
    };
    this.getQuantytiIncuded = function () {
        return this.quantytiIncuded;
    };
    this.setTyperoom = function (typeroom) {
        this.typeroom = typeroom;
    };
    this.getTyperoom = function () {
        return this.typeroom;
    };
    this.setRenday = function (renday) {
        this.renday = renday;
    };
    this.getRenday = function () {
        return this.renday;
    };
    this.totalPay = function () {
        let money = 0;
        if (this.getTypeService() === "Villa") {
            money = 500;
        } else if (this.getTypeService() === "House") {
            money = 300;
        } else if (this.getTypeService() === "Room") {
            money = 200;
        }
        return this.getRenday() * money * (1 - this.getDiscount() / 100);

    }
}

File: Module2/src/casestudy_module2/models/Customer.java
package casestudy_module2.models;

public class Customer {
    private String name;
    private int birthday;
    private String gender;
    private int cmnd;
    private int sdt;
    private String email;
    private String typeCustomer;
    private String address;
    private String services;

    public Customer(String name, int birthday, String gender, int cmnd, int sdt,
                    String email, String typeCustomer, String address, String services) {
        this.name = name;
        this.birthday = birthday;
        this.gender = gender;
        this.cmnd = cmnd;
        this.sdt = sdt;
        this.email = email;
        this.typeCustomer = typeCustomer;
        this.address = address;
        this.services = services;
    }

    public Customer() {
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getBirthday() {
        return birthday;
    }

    public void setBirthday(int birthday) {
        this.birthday = birthday;
    }

    public String getGender() {
        return gender;
    }

    public void setGender(String gender) {
        this.gender = gender;
    }

    public int getCmnd() {
        return cmnd;
    }

    public void setCmnd(int cmnd) {
        this.cmnd = cmnd;
    }

    public int getSdt() {
        return sdt;
    }

    public void setSdt(int sdt) {
        this.sdt = sdt;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getTypeCustomer() {
        return typeCustomer;
    }

    public void setTypeCustomer(String typeCustomer) {
        this.typeCustomer = typeCustomer;
    }

    public String getAddress() {
        return address;
    }

    public void setAddress(String address) {
        this.address = address;
    }

    public String getServices() {
        return services;
    }

    public void setServices(String services) {
        this.services = services;
    }
    public void showInfor(){
        System.out.println("Name:"+ this.name+"Birthday:"+ this.birthday+"Gender:"+ this.gender+"Cmnd:"+ this.cmnd
        +"Phone:"+this.sdt+"Email:"+ this.email+"Type Customer:"+ this.typeCustomer+"Address:"+this.address+"Services:"+ this.services);

    }


    @Override
    public String toString() {
        return "Customer{" +
                "name='" + name + '\'' +
                ", birthday=" + birthday +
                ", gender='" + gender + '\'' +
                ", cmnd=" + cmnd +
                ", sdt=" + sdt +
                ", email='" + email + '\'' +
                ", typeCustomer='" + typeCustomer + '\'' +
                ", address='" + address + '\'' +
                ", services='" + services + '\'' +
                '}';
    }
}
